app/assets/javascripts/application.js

```diff
@@ -6,11 +6,15 @@
 //= require length_counter
 //= require markdown_preview
 //= require toggle_display_with_checked_input
-
+//= require paste_html_to_govspeak
 jQuery(function ($) {
   $('.js-length-counter').each(function () {
     new GOVUK.LengthCounter({ $el: $(this) }) // eslint-disable-line no-new
   })
 
+  $('.js-paste-html-to-govspeak').each(function () {
+    new GOVUK.PasteHtmlToGovspeak({ $el: $(this) }) // eslint-disable-line no-new
```

Comment:
As per previous comment I'm not sure why you've created a new class for essentially one line of code: 

```
element.$el[0].addEventListener('paste', window.pasteHtmlToGovspeak.pasteListener)
```

we may as well have this in here I'd have thought, and can have it as: `this.addEventListener('paste', window.pasteHtmlToGovspeak.pasteListener)`